scripts/manage-index.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  createTestIndex,
+  deleteTestIndex,
+  getClusterIndicies,
+  getLocalIndicies,
+  updateTestIndex,
+} from '../services/index-management'
+
+import inquirer from 'inquirer'
+import { type } from 'node:os'
+
+async function main() {
+  const { action } = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'action',
+      message: 'What do you want to do?',
+      choices: [
+        'Create a new index',
+        'Update an existing index',
+        'Delete an existing index',
+      ],
+    },
+  ])
+
+  if (action === 'Create a new index') {
+    const local = await getLocalIndicies()
+    const cluster = await getClusterIndicies()
+    // should only be able to update indexes which exist locally and NOT in cluster
```

Comment:
I think this comment is wrong? 
```suggestion
    // should only be able to create indices which don't exist
```

But this check feels superfluous, we could just error from elastic as it that's what elastic checks.

The check I image we're looking for is that we search for a `ccr--` follower index that we can create from? But that should be a secondary step.